python_modules/dagster/dagster/_core/storage/event_log/sql_event_log.py

```diff
@@ -1126,6 +1213,22 @@ def _add_assets_wipe_filter_to_query(
 
         return query
 
+    def get_asset_event_tags(self, asset_key: AssetKey) -> Sequence[Tuple[str, AbstractSet[str]]]:
+        tags = defaultdict(set)
+        query = (
+            db.select([AssetEventTagsTable.c.key, AssetEventTagsTable.c.value])
+            .distinct(AssetEventTagsTable.c.key, AssetEventTagsTable.c.value)
+            .where(AssetEventTagsTable.c.asset_key == asset_key.to_string())
+        )
+
+        with self.index_connection() as conn:
+            results = conn.execute(query).fetchall()
+
+        for key, value in results:
+            tags[key].add(value)
+
+        return sorted(list([(k, v) for k, v in tags.items()]), key=lambda x: x[0])
```

Comment:
Is this the right signature?  Will this power the status queries where one dimension is held fixed?